Formula/virgil.rb

```diff
@@ -13,22 +13,16 @@ class Virgil < Formula
     sha256 "ab03a53e7f602341811f1ce6af9ff6b41ccf84ca879f59916fe4f59ff3baca64" => :high_sierra
   end
 
-  depends_on "dep" => :build
   depends_on "go" => :build
+  depends_on :xcode => build
```

Comment:
Why?

Reply:
For use xcode make

Reply:
Changed by advice from @Bo98 :
```
Unless it strictly needs a newer make, can we remove this and let Xcode make be used? Best to avoid adding dependencies unless it's needed.
```

Reply:
Any advice on how can I specify that my formula depends on Xcode command line tools, not on full Xcode?

Reply:
Either will work. Homebrew will manage it for you and you can assume `make` will be available. Apologies for the confusion.